packages/core/src/anchor/ethereum/ethereum-anchor-service.ts

```diff
@@ -171,7 +171,7 @@ export class EthereumAnchorService implements AnchorService {
     const requestUrl = [this.requestsApiEndpoint, tip.toString()].join('/')
     const cidStream = { cid: tip, streamId }
```

Comment:
note I refactored this function in https://github.com/ceramicnetwork/js-ceramic/commit/ee1370a6181fd75391c3e47a86eacb4f79caff07.

Probably will want to double-check how the merge goes to make sure git handles it right